lib/Rider.dart

```diff
@@ -9,15 +9,15 @@ class Rider {
   String phoneNumber;
   String firstName;
   String lastName;
-  Map<String, dynamic> accessibilityNeeds;
+  List<String> accessibilityNeeds;
 
   Rider(this.id, this.email, this.phoneNumber, this.firstName, this.lastName, this.accessibilityNeeds);
 
   factory Rider.fromJson(Map<String, dynamic> json) {
-    return Rider(json['id'], json['email'], json['phoneNumber'], json['firstName'], json['lastName'], Map<String, dynamic>.from(json['accessibilityNeeds']));
+    return Rider(json['id'], json['email'], json['phoneNumber'], json['firstName'], json['lastName'], List.from(json['accessibility']));
   }
 
-  String accessibilityString() {
+  /*String accessibilityString() {
```

Comment:
Why the commenting out instead of removing?

Reply:
whoops, a remnant from before accessibility changed in backend--removed